public/app/features/explore/QueryLibrary/QueryTemplateForm.tsx

```diff
@@ -29,11 +29,6 @@ export type QueryDetails = {
   description: string;
 };
 
-const VisibilityOptions = [
```

Comment:
`t()` should not be used in top level context as it needs initialization so moved.